lib/src/config/theme/app_themes.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+abstract class AppTheme {
+  static ThemeData get dark {
+    return ThemeData(
+      colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
+      useMaterial3: true,
+    );
+  }
+}
```

Comment:
EOF. Configure your IDE to automatically add a line feed at the end of the file

Reply:
Fixed